react-scanner.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	// Crawl the entire repo
+	crawlFrom: './',
+	// Needed for properly reporting components with dot notation
+	includeSubComponents: true,
+	// Exclude usage in tests and stories.
+	globs: [ '**/!(test|stories)/!(*stories).@(js|ts)?(x)' ],
+	// Exclude any vendor or docs directories
+	exclude: [ 'docs', 'jetpack_vendor', 'node_modules', 'tools', 'vendor' ],
```

Comment:
May want to add `'wordpress'` here too, to exclude the copies of WordPress installed by WorDBless.